changelog/19077.txt

```diff
@@ -0,0 +1,6 @@
+```release-note:feature
+auth/azure: Adds support for rotate root to Azure Auth
```

Comment:
Wasn't sure if I should list all the other features added or just the latest one. Thoughts?

Reply:
I think if they are all being added to 1.13 and there's no previous changelog entry for them, it makes sense to add them all now.

Reply:
I'd say list all of the features/improvements/fixes that are being brought in at this time. I'd say you can leave the dependency upgrade entry out of the Vault changelog though.

Reply:
Also, there is a specific format for "feature" changelog entries. I don't believe these conform to the format.

Reply:
Ah thanks for the clarification, @austingebauer! Updated the changelog to match the desired format for multiple features